composer/trainer/trainer.py

```diff
@@ -1343,6 +1348,9 @@ def fit(
             _validate_precision(precision, self._device, self.deepspeed_enabled)
             self.state.precision = precision
 
+            # update scaler since precision was provided
+            self.state.scaler = ClosureGradScaler() if self._use_closures() else GradScaler()
```

Comment:
In addition, if `reset_time` is set, would it make sense to also wipe the grad scaler state?

Reply:
That sounds unnecessary.